source/_components/media_player.cambridgeaudio.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: page
+title: "Cambridge Audio (StreamMagic)"
+description: "Instructions on how to integrate Cambridge Audio network audio players (using their StreamMagic platform) into Home Assistant."
+date: 2018-05-10 09:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: cambridgeaudio.png
+ha_category: Media Player
+ha_iot_class: "Local Polling"
+ha_release: 0.68
+---
+
+The `cambridgeaudio` platform allows you to control Cambridge Audio [network audio players](https://www.cambridgeaudio.com/usa/en/products/hi-fi/network-players) that are based on their _StreamMagic_ platform.
+
+Verified to work with:
+
+- Azur 851N
+
+If your non-listed device works with this platform, if you encounter any bugs, or want to request a specific feature, please feel free to [report them upstream](https://github.com/sebk-666/stream_magic)
+
+A few notes:
+
+- The platform currently does not support the pre-amp features of the player, i.e., no controls for volume, muting, balance and such are currently available.
+- The sources menu lets you choose between the (Internet radio) presets you set your player up with. It's currently not possible to switch the actual digital inputs with this platform.
+
+
+## {% linkable_title Configuration %}
+
+The platform will be loaded automatically by discovery component. If you want to configure it manually, add the following to your `configuration.yaml` file:
+
+```yaml
+media_player:
+  - platform: cambridgeaudio
+    name: CA851N
+    host: 192.168.1.31
+    poweroff_command: OFF
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

Reply:
Thanks for reviewing! I've made the requested changes.